### PR TITLE
Update instructions on how to deal with a fire-y LiPo

### PR DESCRIPTION
**Pull request by Scarzy**
Based on feedback from Surrey Fire & Rescue Service

### CODE REVIEW

**Review by trickeydan — APPROVED** [review]

File: docs/kit/event-operations/battery-charging.md [hunk 1]
```diff
@@ -43,7 +43,14 @@ Every volunteer performing the Battery Charging role must have read this documen
...
 
-The batteries themselves have many built-in safety features and, unless abused, are completely safe. All batteries undergoing charging must be placed inside a fireproof charging bag. In the extremely unlikely event of a fire due to a battery DO NOT attempt to tackle the fire - the batteries contain a flammable metal (Lithium) which cannot be safely tackled by most fire extinguishers.
+The batteries themselves have many built-in safety features and, unless abused, are completely safe. All batteries undergoing charging must be placed inside a fireproof charging bag. In the extremely unlikely event of a fire due to a battery:
+- DO NOT use a fire extinguisher - the batteries contain hazardous materials which cannot be safely tackled by most fire extinguishers.
+- DO NOT open or attempt to remove anything from the charging bag.
+ONLY IF SAFE TO DO SO:
```
Comment (WillB97) [inline docs/kit/event-operations/battery-charging.md:49]:
```suggestion

ONLY IF SAFE TO DO SO:
```
I assume the intention is for these to be separate lists

**Comment by TomWheal** [comment]
You can buy a fireproof box designed for bad lipos. Might be worth us getting one to put lipos in the bag inside in case of an emergency. 

https://uk.rs-online.com/web/p/battery-storage-boxes/2000580
(I know it's pricey)

File: docs/kit/event-operations/battery-charging.md (hunk 1 above)
Comment (Scarzy) [reply docs/kit/event-operations/battery-charging.md:49]: It is a continuation of the list "In the extremely unlikely event of a fire due to a battery:", but with the extra qualifier of only if safe, so I'm not sure if the extra line is helpful or confusing?

**Comment by Scarzy** [comment]
> You can buy a fireproof box designed for bad lipos. Might be worth us getting one to put lipos in the bag inside in case of an emergency.

The intent of these instructions is for use when tackling one actually on fire, and is based on feedback from a fire and rescue service.
These boxes seem designed for transporting lipos that aren't fire-y, but not far off, and containing a fire should one break out. I suspect there are specific instructions on how to pack a battery in there, and it isn't just "throw and slam".
I'm really not comfortable with the idea of suggesting we know better than a fire and rescue service and in doing so using something in a manner it doesn't seem designed for...

Maybe there is an argument for getting one of those boxes, but I'm not sure it's applicable to this PR.

File: docs/kit/event-operations/battery-charging.md (hunk 1 above)
Comment (WillB97) [reply docs/kit/event-operations/battery-charging.md:49]: Without the new line the heading is appended to the last line of the previous list.

**Pull request merged**